ansible/inventory.yml

```diff
@@ -14,22 +14,18 @@ hosts:
         ubuntu1604-x64-1: {ip: 138.197.224.240, alias: www}
         ubuntu1804-x64-1: {ip: 178.128.202.158, alias: gzemnid}
         ubuntu1804-x64-2: {ip: 45.55.45.227, alias: unofficial-builds}
-        ubuntu1804-x64-3: {ip: 157.245.7.159, alias: metrics}
-
-    - ibm:
-        ubuntu1804-x64-1: {ip: 169.45.166.50, alias: ci-release}
-        ubuntu2004-x64-1: {ip: 169.60.150.91, alias: ansible}
```

Comment:
Could you check the large number of changes/deletions of non-arm64 hosts? 🙂

Reply:
Sorted - my rebase I'd done didn't seem to have the desired effect for some reason ...